src/main/java/org/reaktivity/nukleus/amqp/internal/stream/AmqpServerFactory.java

```diff
@@ -3234,7 +3242,11 @@ private void onDecodeTransfer(
                             this.messageFormat = messageFormat;
                         }
                     }
-                    if (!more)
+                    if (aborted)
+                    {
+                        flags = FLAG_INCOMPLETE;
+                    }
+                    if (!more && !aborted)
```

Comment:
Suggest leaving a blank line between successive `if` blocks so they are not easily misread as `if-else`.